src/view/resolvers/ExpressionProxy.js

```diff
@@ -110,6 +94,33 @@ export default class ExpressionProxy extends Model {
 		this.handleChange();
 	}
 
+	bind () {
+		this.resolvers = [];
+		this.models = this.template.r.map( ( ref, index ) => {
+			const model = resolveReference( this.fragment, ref );
+			let resolver;
+
+			if ( !model ) {
+				resolver = this.fragment.resolve( ref, model => {
+					removeFromArray( this.resolvers, resolver );
+					this.models[ index ] = model;
+					this.bubble();
+				});
+
+				this.resolvers.push( resolver );
+			}
+
+			return model;
+		});
+
+		this.bubble();
+	}
+
+	rebind () {
+		this.unbind();
+		this.bind();
```

Comment:
Under what conditions does the `rebind` for an expression happen? Is it when it is used within a section?